php.ini-production

```diff
@@ -1764,20 +1764,20 @@ ldap.max_links = -1
 
 [opcache]
 ; Determines if Zend OPCache is enabled
-;opcache.enable=0
+;opcache.enable=1
 
 ; Determines if Zend OPCache is enabled for the CLI version of PHP
-;opcache.enable_cli=0
+;opcache.enable_cli=1
```

Comment:
I didnt found anything which declares the opcache on cli as experimental or something. Also didnt find any downsides which this might have, therefore I changed the default to 1.

If this is a problem, we can change this setting back to 0.
In case we find a reason why one should disable this setting we should document it in this ini-file though.